### PR TITLE
add models for ALB instances to migrator

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:

- add models for ALB instances to migrator

## Security considerations

None

### CODE REVIEW

File: migrator/models.py [hunk 1]
```diff
@@ -85,3 +85,54 @@ class CdnCertificate(CdnBase):
...
     expires = sa.Column(postgresql.TIMESTAMP)
+
+
+class CustomDomainRoute(CustomDomainBase):
+    __tablename__ = "routes"
+
+    guid = sa.Column(sa.Text, primary_key=True)
+    state = sa.Column(sa.Text)
+    domains = sa.Column(sa.Text)
+    challenge_json = sa.column(postgresql.BYTEA)
```
Comment (apburnes) [inline migrator/models.py:96]: Is there a difference between `sa.Column` and `sa.column`?

**Review by apburnes — COMMENTED** [review]
Just a question on the `sa.column`.

File: migrator/models.py (hunk 1 above)
Comment (bengerman13) [reply migrator/models.py:96]: yeah,`sa.column` is a typo

File: migrator/models.py [hunk 2]
```diff
@@ -85,3 +85,54 @@ class CdnCertificate(CdnBase):
...
+class CustomDomainRoute(CustomDomainBase):
+    __tablename__ = "routes"
+
+    guid = sa.Column(sa.Text, primary_key=True)
+    state = sa.Column(sa.Text)
+    domains = sa.Column(sa.Text)
+    challenge_json = sa.Column(postgresql.BYTEA)
+    user_data_id = sa.Column(sa.Integer)
+    alb_proxy_arn = sa.Column(sa.Text)
+    certificates = orm.relationship("CustomDomainCertificate")
```
Comment (ccostino) [inline migrator/models.py:99]: No need/desire for [`back_populates`](https://docs.sqlalchemy.org/en/14/orm/relationship_api.html#sqlalchemy.orm.relationship.params.back_populates) here?  Totally fine if not, was just curious.

**Review by ccostino — APPROVED** [review]
Thanks, @bengerman13!  One quick question, but this looks good to me.

**Comment by bengerman13** [comment]
fixes #74

File: migrator/models.py (hunk 2 above)
Comment (bengerman13) [reply migrator/models.py:99]: I think that's handled on 110

**Pull request merged**